test/AutoDiff/simple_math.swift

```diff
@@ -73,13 +73,6 @@ SimpleMathTests.test("CaptureGlobal") {
   expectEqual(30, gradient(at: 0, in: foo))
 }
 
-let foo: (Float) -> Float = { x in
```

Comment:
Move global closure test to `test/AutoDiff/autodiff_diagnostics.swift`?

Reply:
I'm going to do a more comprehensive patch to disable flow-sensitivity for AD function conversion. PR coming soon.